### PR TITLE
Merge 2.3 master

### PR DESCRIPTION
**Pull request by hypeJunction**
Conflicts:
composer.json
composer.lock
install/cli/travis_installer.php

### CODE REVIEW

**Review by jdalsem — APPROVED** [review]

**Comment by hypeJunction** [comment]
Looks like Travis wasn't running plugin simpletests. This has surfaced some issues with the group plugin

**Comment by hypeJunction** [comment]
@jdalsem could you merge, I am doing more work on master

File: engine/classes/ElggPlugin.php [hunk 1]
```diff
@@ -904,7 +904,7 @@ protected function includeFile($filename) {
 		}
 
 		try {
-			$ret = require_once $filepath;
+			$ret = include $filepath;
```
Comment (jdalsem) [inline engine/classes/ElggPlugin.php]: uh isn't this causing the same problem on 2.3??

**Comment by jdalsem** [comment]
> Looks like Travis wasn't running plugin simpletests.

do we have this problem also on 2.3?

**Comment by jdalsem** [comment]
i am feeling not so confident that the require_once change in 2.3 is without a problem

**Comment by hypeJunction** [comment]
The problem is with new elgg-plugin files. In 2.3 we only included start, activate and deactivate

**Comment by hypeJunction** [comment]
Not sure if we have the tests problem on 2.3. For have time nor desire to look

**Comment by jdalsem** [comment]
can we remove the change in 2.3 and only apply it to master? It was not needed for the seeding perse am i right?

**Comment by hypeJunction** [comment]
Tests were failing because plugin files were included multiple times. If you can hunt down the issue, it can be removed. But I am not going to spend time on it. It happened before locally as well.

**Comment by jdalsem** [comment]
i thought the require_once was reverted when upmerging, now i see that it was indeed only added to the extra/new include. Merging this now

**Pull request merged**